006-vortexa-tech-challenge/src/header.tsx

```diff
@@ -0,0 +1,17 @@
+import { AppBar, Box, Toolbar, Typography } from "@mui/material";
+
+import React from "react";
```

Comment:
Similarly here, sort the imports.

```suggestion
import React from "react";
import { AppBar, Box, Toolbar, Typography } from "@mui/material";
```